add pronounciation on the review page at the end of the exercises

beamb opened the pull request:
Resolves #35 and styled all buttons throughout exercises.

mircealungu commented:
also, the speech button is more tall than wide; that's a bit unusual; 
maybe we cold make it the same size as the "next" button? 
![image](https://user-images.githubusercontent.com/464519/117547360-e2d17600-b037-11eb-8b40-5edb7e9dfee6.png)

mircealungu commented:
that being said, the animation on the buttons is awesome :) 
you have good taste in UIs :)

beamb commented:
Agreed! I've changed all buttons to be on an orange background and with a heavier font weight. The contrast is still fairly poor but I guess we should change the styling throughout the app to solve this issue.

I've removed the "react-promise-tracker", hence the merge-conflict in `package-lock.json`. But I've spotted a second conflict which I'm unsure how to resolve. 

![image](https://user-images.githubusercontent.com/60703644/117551133-714bf480-b044-11eb-9a3d-91ce7f2c6243.png)

Does `resolved` need to be empty and without `integrity` SHA or is it ok to go with the newer version?

mircealungu commented:
Strange the conflict!

Is it my fault? I've pushed a new version of packages-lock.json after running an audit-fix to your branch.

On my side I see: 

![image](https://user-images.githubusercontent.com/464519/117562045-9e2ff400-b0a4-11eb-9b97-43585ede7f53.png)

beamb commented:
Ok. Strange. But I've fixed it now so unless any other changes are needed, this branch is merge ready 😁

mircealungu commented:
@beamb: do you also see this on your side? 
![image](https://user-images.githubusercontent.com/464519/117585766-cd824780-b11c-11eb-8133-e445478c431e.png)

mircealungu commented:
Ah! It's again my browser font settings: 

![image](https://user-images.githubusercontent.com/464519/117585881-834d9600-b11d-11eb-9d39-f9dc09f5102e.png)

but because I'm blind as a bat... we shouldn't spent more time on this I think.  I'll change my settings for now. if I change the font to normal, then all is well again!

![image](https://user-images.githubusercontent.com/464519/117585907-ad9f5380-b11d-11eb-9d85-cd314d46c0fa.png)
